application-development/cloud-native/README.md

```diff
@@ -6,6 +6,38 @@ The intended purpose is to provide a clear and well-defined insight into the con
 
 Customers and partners developing applications and thinking of moving into the cloud can find anything needed here, from inspiring slides to samples of code.
 
+# Table of Contents
+ 
+1. [Team Publications](#team-publications)
+2. [Useful Links](#useful-uinks)
+
+## Team Publications
+
+## Useful Links
+
+## Hyperscaler translation
```

Comment:
Can you remove that from this page and put it as a separate asset?

Create a folder underneath this page hyper-scaler-translation. Add a README.md as per the template (see confluence) and add a LICENSE file. Move this Translation to a dedicated MD file under hyper-scaler-translation/files